compose/ui/ui/src/webTest/kotlin/androidx/compose/ui/SelectionContainerTests.kt

```diff
@@ -44,6 +45,13 @@ import org.w3c.dom.events.MouseEventInit
 
 class SelectionContainerTests : OnCanvasTests {
 
+    @BeforeTest
+    fun setup() {
+        // We should use this method whenever we are relying on the fact that document coordinates start exactly at (0, 0)
+        // TODO: strictly speaking, this way one test suit affects the other in that sense that styles can be injected by different tests suite
+        OnCanvasTests.injectDefaultStyles()
+    }
```

Comment:
Maybe we can call it from fun composableContent?
So no need to think about calling it  when writing the tests?

Since once it's injected, it affects all tests. So until it becomes a problem for us, we can simplify the test setup?

Reply:
I agree, that's a fair note

Reply:
@eymar pushed